Source/Particles/MultiParticleContainer.H

```diff
@@ -59,9 +59,11 @@ public:
 
     ~MultiParticleContainer() {}
 
-    WarpXParticleContainer& GetParticleContainer (int ispecies) {
-        return *allcontainers[ispecies];
-    }
+    WarpXParticleContainer&
+    GetParticleContainer (int ispecies) const {return *allcontainers[ispecies];}
+
+    WarpXParticleContainer*
+    GetParticleContainerPtr (int ispecies) const {return allcontainers[ispecies].get();}
```

Comment:
Do we use both functions?

Reply:
Yes, one in the old diags, one in the new. One will be deleted in another PR, when we get rid of the old ones.